### ISSUE TITLE
[GENERAL] Vicerators dirty floors as they fly.

### ISSUE THREAD

**Issue by VictorZisthus** [CHOMPStation2/CHOMPStation2#4674]
#### Brief description of the bug
Title.

Mainly posting this here for me to fix later if someone else does not beat me to it.

**Comment by CMDRYouth** [comment]
A bug?  They're miniature helicopters!  They're dusting up the ground and floor, spreading dirt and debris all over the place.
I'd be surprised if they didn't spread clouds of dust everywhere they flew.

**Comment by VictorZisthus** [comment]
Let me rephrase.

They CREATE dirt as they move. They don't just spread it around.

**Comment by lbnesquik** [comment]
Stale, but this is true.
